src/main/java/digital/slovensko/autogram/model/AutogramDocument.java

```diff
@@ -0,0 +1,38 @@
+package digital.slovensko.autogram.model;
+
+import eu.europa.esig.dss.model.DSSDocument;
+
+public class AutogramDocument {
+    private final DSSDocument document;
+
+    private char[] openDocumentPassword = new char[0];
+    private char[] masterPassword = new char[0];
+
+    public AutogramDocument(DSSDocument document) {
+        this.document = document;
+    }
+
+    public char[] getOpenDocumentPassword() {
+        return openDocumentPassword;
+    }
+
+    public void setOpenDocumentPassword(char[] openDocumentPassword) {
+        this.openDocumentPassword = openDocumentPassword;
+    }
+
+    public char[] getMasterPassword() {
+        return masterPassword;
+    }
+
+    public void setMasterPassword(char[] masterPassword) {
+        this.masterPassword = masterPassword;
+    }
+
+    public char[] getSigningPassword() {
+        return openDocumentPassword.length > 0 ? openDocumentPassword : masterPassword;
+    }
+
+    public DSSDocument getDocument() {
```

Comment:
veľa krát sa používa v kóde `job.getDocument().getDocument()`, navrhujem túto metódu nazvať `getDSSDocument()`, čo myslíš? @xhyrom